docs/modules/cataloging/pages/batch_importing_MARC.adoc

```diff
@@ -57,7 +57,7 @@ set for authority records or bibliographic records.
 
 . Click *Save*.
 +
-image::media/Batch_Importing_MARC_Records1.jpg[Batch_Importing_MARC_Records1]
+image::batch_importing_MARC/new_record_match.jpg
```

Comment:
These image macros will need to end with a set of square brackets, per the [antora documentation](https://docs.antora.org/antora/2.2/asciidoc/insert-image/#block-image-macro-structure).  Better still would be to add some alt text into those brackets, for example:

    image::batch_importing_MARC/new_record_match.jpg["the new record match form, where a user has entered a match set type, name, and owning library for their new record matching definition set"]

Reply:
I tried doing a push to origin via GitHub desktop.  Let me know if you received those.  If not, I'll try again.